bracketed-paste.el

```diff
@@ -128,7 +128,8 @@
   '(suspend-hook suspend-tty-functions kill-emacs-hook delete-frame-functions))
 
 (defun bracketed-paste--safe-tty-state-call (terminalish send-tty)
-  (cond ((null terminalish) (funcall send-tty))
+  (cond ((window-system) nil) ; don't call send-tty if not a TTY
```

Comment:
Yeah, I just wasn't sure since when it was deprecated as I'm using Emacs 25.

Furthermore, the doc for `window-system` states:

> The value is a symbol:
> nil for a termcap frame (a character-only terminal),
> ‘x’ for an Emacs frame that is really an X window,
> ‘w32’ for an Emacs frame that is a window on MS-Windows display,
> ‘ns’ for an Emacs frame on a GNUstep or Macintosh Cocoa display,
> ‘pc’ for a direct-write MS-DOS frame.

So, I suspect that `display-graphic-p` would return `nil` for the `pc` case above and this is not what I want. I truly want to run this only on termcap devices.